### ISSUE TITLE
Add Microsoft Credentials Docker Implementation 

### ISSUE THREAD

**Issue by OR13** [decentralized-identity/JWS-Test-Suite#8]
blocked by https://github.com/decentralized-identity/JWS-Test-Suite/issues/7

**Comment by OR13** [comment]
We would hope that other developers can help assist with this, but I am happy to help coordinate and assist.

**Comment by OR13** [comment]
we did this, its rough but its there, open a new issue for enhancements if required.
